static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -60,175 +301,120 @@ export interface UsageStatsOrganizationProps extends WithRouterProps {
+        | 'card-rate-limited'
+        | 'card-invalid'
+        | 'chart-title'
+    ) => void;
+    orgStats: UseApiQueryResult<UsageSeries | undefined, RequestError>;
+    usageChart: React.ReactNode;
+  }) => React.ReactNode | React.ReactNode;
   clientDiscard?: boolean;
   clock24Hours?: boolean;
+  endpointQuery?: ReturnType<typeof getEndpointQuery>;
+  projectDetails?: React.ReactNode[];
 }
 
-type UsageStatsOrganizationState = {
-  orgStats: UsageSeries | undefined;
-  metricOrgStats?: UsageSeries | undefined;
-} & DeprecatedAsyncComponent['state'];
-
-/**
- * This component is replaced by EnhancedUsageStatsOrganization in getsentry, which inherits
- * heavily from this one. Take care if changing any existing function signatures to ensure backwards
- * compatibility.
- */
-class UsageStatsOrganization<
-  P extends UsageStatsOrganizationProps = UsageStatsOrganizationProps,
-  S extends UsageStatsOrganizationState = UsageStatsOrganizationState,
-> extends DeprecatedAsyncComponent<P, S> {
-  componentDidUpdate(prevProps: UsageStatsOrganizationProps) {
-    const {
-      dataDatetime: prevDateTime,
-      projectIds: prevProjectIds,
-      dataCategoryApiName: prevDataCategoryApiName,
-    } = prevProps;
-    const {
-      dataDatetime: currDateTime,
-      projectIds: currProjectIds,
-      dataCategoryApiName: currentDataCategoryApiName,
-    } = this.props;
-
-    if (
-      prevDateTime.start !== currDateTime.start ||
-      prevDateTime.end !== currDateTime.end ||
-      prevDateTime.period !== currDateTime.period ||
-      prevDateTime.utc !== currDateTime.utc ||
-      prevDataCategoryApiName !== currentDataCategoryApiName ||
-      !isEqual(prevProjectIds, currProjectIds)
-    ) {
-      this.reloadData();
-    }
-  }
-
-  getEndpoints(): ReturnType<DeprecatedAsyncComponent['getEndpoints']> {
-    return [['orgStats', this.endpointPath, {query: this.endpointQuery}]];
-  }
-
-  /** List of components to render on single-project view */
-  get projectDetails(): React.JSX.Element[] {
-    return [];
-  }
-
-  get endpointPath() {
-    const {organization} = this.props;
-    return `/organizations/${organization.slug}/stats_v2/`;
+export type CardMetadata = Record<
+  'total' | 'accepted' | 'filtered' | 'rateLimited' | 'invalid',
+  {
+    title: React.ReactNode;
+    help?: React.ReactNode;
+    isEstimate?: boolean;
+    score?: string;
+    trend?: React.ReactNode;
   }
+>;
+
+function UsageStatsOrganization({
+  dataDatetime,
+  projectIds,
+  dataCategoryApiName,
+  organization,
+  dataCategory,
+  dataCategoryName,
+  clientDiscard,
+  handleChangeState,
+  chartTransform,
+  children,
+  endpointQuery,
+}: UsageStatsOrganizationProps) {
+  const router = useRouter();
+  const orgStatsQuery =
+    endpointQuery ??
+    getEndpointQuery({
+      dataCategoryApiName,
+      dataDatetime,
+      organization,
+      projectIds,
+      endpointQueryDatetime: getEndpointQueryDatetime(dataDatetime),
+    });
```

Comment:
this probably needs to be memoized if its being used as a dependency later since it will  create a new object every render